addons/mail/static/src/core/common/suggestion_service.js

```diff
@@ -101,6 +104,19 @@ export class SuggestionService {
         this.store.insert(data);
     }
 
+    /**
+     * @param {string} term
+     * @param {import("models").Thread} [thread]
+     */
+    async fetchRoles(term) {
+        const data = await this.orm.silent.searchRead(
```

Comment:
Better make it a param of `/mail/data` so it can potentially be batched with other requests